src/main/scala/ru/org/codingteam/horta/Application.scala

```diff
@@ -20,12 +20,17 @@ import ru.org.codingteam.horta.plugins.markov.MarkovPlugin
 import ru.org.codingteam.horta.plugins.pet.PetPlugin
 import ru.org.codingteam.horta.plugins.visitor.VisitorPlugin
 import ru.org.codingteam.horta.plugins.wtf.WtfPlugin
-import ru.org.codingteam.horta.plugins.{VersionPlugin, AccessPlugin, FortunePlugin}
+import ru.org.codingteam.horta.plugins.{AccessPlugin, FortunePlugin, VersionPlugin}
 import ru.org.codingteam.horta.protocol.jabber.JabberProtocol
 import scalikejdbc.GlobalSettings
 
 object Application extends App with StrictLogging {
 
+  //TODO: This should be eventually handled by some kind of dependency injeciton
+  val eventEndpoints = List(
+    new TwitterEndpoint()
```

Comment:
You should also disable `TwitterEndpoint` (or, better, even don't add it here) if Twitter isn't configured.